fastlane/lib/fastlane/actions/app_store_build_number.rb

```diff
@@ -30,7 +30,7 @@ def self.run(params)
             if testflight_version
               version_number = testflight_version
             else
-              version_number = UI.input("You have to specify a new version number")
+              version_number = UI.input("You have to specify a new version number, as there is multiple to choose from")
```

Comment:
Should be "as there *are* multiple to choose from" 👍

Reply:
Yep, nice thanks